Tech Lead/Minutes/20231010.md

```diff
@@ -0,0 +1,55 @@
+## Technical Lead Call 10 October 2023 (online; special - ESMValCore=2.10 release)
+
+## Participants
+- Attendees: @ehogan @valeriupredoi @bouweandela @schlunma @zklaus
+- Absentees: @sloosvel
+- Chairs and minutes takers: @valeriupredoi
+
+  - Names legend:
+    - Bouwe Andela [@bouweandela](https://github.com/bouweandela)
+    - Klaus Zimmermann [@zklaus](https://github.com/zklaus)
+    - Saskia Sloosveldt [@sloosvel](https://github.com/sloosvel)
+    - Valeriu Predoi [@valeriupredoi](https://github.com/valeriupredoi)
+    - Manuel Schlund [@schlunma](https://github.com/schlunma)
+    - Emma Hogan [ehogan](https://github.com/ehogan)
+
+## Pending Action Items
+
+| Meeting | Assignees | Item | Status |
+|-|-|-|-|
+|2023.07|@ehogan|KGO recording solutions|See [ESMValTool: discussion: #3247](https://github.com/ESMValGroup/ESMValTool/discussions/3247) for the latest news|
+|2023.07|@bouweandela, @remi-kazeroni|Asking the community to check the output once for all| @ehogan: see [ESMValTool: discussion: #3247](https://github.com/ESMValGroup/ESMValTool/discussions/3247) for some additional thoughts on this|
+|2023.07|@valeriupredoi|Shared gmail account to handle to Zenodo credentials|pending|
+|2023.09|@schlunma|Talk about ERA5|pending|
+
+## Proposed topics for discussion (to be ordered then added to the Agenda section at the start of the meeting)
```

Comment:
I was wondering (since this is what @zklaus did in #129) whether it would be better not to include the `Proposed topics for discussion`, since many of them weren't discussed. What do you think?

Reply:
sounds good! Let me nuke them off here, cheers @ehogan :beer:

Reply:
done in https://github.com/ESMValGroup/Community/pull/130/commits/ccfcff2c20a31de6dadab8591fda0cb689c7a2b4